[IMP] barcode: add packages to receipts and deliveries (#18004)
odoo/documentation: content/applications/inventory_and_mrp/barcode/operations/receipts_deliveries.rst

The **Barcode** app can be used to process receipts, deliveries, and other types of operations in
real time using a barcode scanner or the Odoo mobile app.

> This makes allows operations on the warehouse floor to be processed in real time, rather than

Reviewer: ```suggestion
This allows operations on the warehouse floor to be processed in real time, rather than
```

waiting to validate transfers from a computer. This real-time processing ensures barcodes are
accurately assigned to the correct products, packages, packaging, transfers, locations, and more,

> reducing errors and keeping inventory data in sync.

Reviewer: ```suggestion
accurately assigned to the correct products, packages, packaging, transfers, locations, and more.
This helps reduce errors and keeps inventory data in sync.
```

Lotsa commas.

From this screen, an overview of all receipts to process within that transfer (**WH/IN/000XX**) is
shown. At the bottom of the screen, there are options to :guilabel:`Add Product` or
:guilabel:`Validate`, depending on whether products need to be added to the operation or if the
whole operation should be validated at once. If the *Packages* setting is enabled in the Inventory
settings, the :guilabel:`Put in Pack` button is also available. If :doc:`quality checks
<../../quality/quality_management/quality_checks>` are required for a product, the
:guilabel:`Quality Checks` button displays.

.. image:: receipts_deliveries/receipts-deliveries-scanner-overview.png
   :alt: Overview of receipts in transfer to scan.

> To process and scan each product individually, scan a product, package, or product packaging, or
> choose a specific product line. Click the :guilabel:`+#` button (for example, `+10`) to indicate

Reviewer: ```suggestion
To process and scan each product individually, choose a specific product line or scan a product, package, or product packaging. Click the :guilabel:`+#` button (for example, `+10`) to indicate
```

Do you think this reads any more clearly? I genuinely can't decide.

:guilabel:`Quantity` line. Change the `0.00` in the line to the desired quantity, or click the
:guilabel:`+#` button (for example, `+12`) to automatically fill the quantity ordered from the |PO|.

Additionally, click the :guilabel:`+1` and :guilabel:`-1` buttons to add or subtract a quantity of
the product. Click the product packaging button (for example, `6-pack`) to add a product packaging.

Below the quantity buttons is the :guilabel:`Destination Location` line, which reads `WH/Stock` by
default, unless another *location* is listed on the product itself. Click this line to reveal a
drop-down menu of additional locations to choose from.

> When the *Packages* feature is enabled in the **Inventory** settings, the following fields appear:

Reviewer: ```suggestion
When the *Packages* feature is enabled in the **Inventory** app's settings, the following fields appear:
```

Suggested just to be thorough.
